app/src/main/res/layout/content_book_detail.xml

```diff
@@ -2,6 +2,7 @@
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/mAddBook"
```

Comment:
The naming convention for layout element ids is @+id/add_book.

Reply:
See [this blog post](https://jeroenmols.com/blog/2016/03/07/resourcenaming/) for xml naming conventions.